frontend/src/pages/Home/HomePage.jsx

```diff
@@ -24,16 +24,16 @@ export const HomePage = () => {
   };
 
   // The logic for this will need to change which might break the tests later on
-  const handleSubmit = async (e) => {
```

Comment:
Just to confirm. The HomePage won't handle the navigation and instead it'll all be handled by the ScrapeRecipe component?

Reply:
yes, because the button that you submit to navigate is on the component of the scrape recipe and I imagine if we add the navbar here the navigate will also be on the navbar component

Reply:
Okok. I was in two minds about whether it should be HomePage or in it's own component. Might need to move or skip some of the tests in HomePage for now so that it doesn't report as failing.